### ISSUE TITLE
[Designer]Bug 37604885: [FlightDetails.Json]: "Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7618]
### Problem Description

[Bug 37604885](https://microsoft.visualstudio.com/OS/_workitems/edit/37604885): [FlightDetails.Json]: "Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card.

Test Environment:
Application: Adaptive Cards
OS version: Dev (OS Build 22499.1010)
Application Version: 3.0.2111.24001
Screen Reader: Narrator

Repro steps:

1. Launch the Adaptive Card application and narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to FlightDetails.Json button by using Tab key and activate it.

4. FlightDetails adaptive card will be appeared on right sight of the pane.

5. Navigate to FlightDetails card by using Tab key.

6. Navigate in scan mode to "Flight to JFK" and observe the issue.

Actual Result:
"Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card. Screen reader is only announcing as, "Flight to JFK".

Expected Result:
"Flight to JFK" should be defined under a heading. Screen reader should announce as, "Flight to JFK heading level 2".

Note: Same issue is also repro with 'Passengers' as it is not defined as heading under 'Flightitinerary' card. (Please refer attachments)

User Experience:
It will be confusing & difficult for the users if Heading will not be defined inside the adaptive card.

Observation: Same issue is also repro with JAWS.

____________Same bug found on Android FC-Jan22 Test Pass____________

Adding <HCL-AdaptiveCards-Android> Tag to this bug.

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version: 2021.10.15.1 (40331737)

Pre-requisite:
Import-Sample Cards->FlightDetails.json; 

Repro Steps:
Turn on the Talkback.
Navigate to the Visual.
Navigate to the sample card.
Swipe up/down with three fingers to change output settings->'headings'
Navigate with up/down with one finger to move to headings present.
Observe that there is no heading present.
Actual Result:
There is no heading provided for the sample card.
Screen reader focus does not move to "FLight to JFK" when navigate in headings settings with up/down (one finger) swipe gesture.

Expected Result:
"Flight to JFK" should be defined as heading in the said sample card programmatically.
Screen reader should announce "FLight to JFK" as a heading and screen reader in 'headings settings' talkback should move to "Flight  to JFK".

Note: 
This issue is repro with 'FlightDetails' sample cards + all the sample configs examples on Android.
Android platform repro attachment added -'Android repro-Flight to JFK is not a heading.mp4'

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Card JSON

```JSON
Also, Repro on UWP
```
